.github/actions/deploy-klt-on-cluster/action.yml

```diff
@@ -2,16 +2,20 @@ name: "Deploy Keptn Lifecycle Toolkit on GH cluster"
 description: "Creates a Kind cluster and deploys Keptn Lifecycle Toolkit"
 inputs:
    cert-manager-version:
-     required: true
+     required: false
      description: "Version of cert-manager that should be deployed"
-   k3d-version:
-     required: true
-     description: "Version of k3d that should be used"
-   k3s-version:
+     # renovate: datasource=github-releases depName=cert-manager/cert-manager
+     default: "v1.10.0"
+   kind-version:
+     required: false
+     description: "Version of kind that should be used"
+     # renovate: datasource=github-releases depName=kubernetes-sigs/kind
+     default: "v0.17.0"
+   k8s-version:
      required: false
-     description: "Version of k3s that should be used"
-     # renovate: datasource=github-releases depName=k3s-io/k3s
-     default: "rancher/k3s:v1.24.8-k3s1"
+     description: "Kubernetes that should be used"
+     # renovate: datasource=github-releases depName=kubernetes/kubernetes
+     default: "v1.26.0"
```

Comment:
i would suggest to default to v1.25 for now since we have all our API dependencies on v1.24